### DISCUSSION TITLE
bugs when I try to run inference_hf.py after finetuning with lora

### DISCUSSION THREAD

**Discussion by RemiAlliah** [zai-org/ChatGLM3#1241] [Q&A]
命令行输入：
 python .\inference_hf.py .\output\isign\checkpoint-3000\ --prompt "今天"周几"
![image](https://github.com/THUDM/ChatGLM3/assets/97057358/ecc7358c-145b-4412-9024-b7b565c47ec3)
![image](https://github.com/THUDM/ChatGLM3/assets/97057358/ac362a26-862d-4b61-ac0b-b5b92b7c6db5)

ValueError: not enough values to unpack (expected 2, got 1)

**Comment by BellyLeung** [comment]
解决了吗？我也碰到了这个问题
